setup.py

```diff
@@ -46,13 +47,14 @@ def find_version(*file_paths):
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
     ],
     keywords='avatar, django',
     author='Eric Florenzano',
     author_email='floguy@gmail.com',
-    maintainer='Grant McConnaughey',
-    maintainer_email='grantmcconnaughey@gmail.com',
-    url='http://github.com/grantmcconnaughey/django-avatar/',
+    maintainer='Luis Ernesto Assandri',
```

Comment:
These changes will need to be reverted before I can merge this PR.

Reply:
Can this not be merged and then the problematic changes to setup.py reverted before pushing a new version to PyPI..?

Reply:
IMO it is not worth having such confusing commits in the git history.  The changes in this PR can be mostly automatically generated with `django-codemod` and `django-upgrade`.  If @luigi370 doesnt fix this PR, @grantmcconnaughey might be able to force-push the fix to the PR branch, otherwise someone else can create this PR without the offending changes.

Reply:
Yep, I'd prefer to not have that change in the commit history. Removing it then squashing the commits down to 1 oughta do the trick.

Reply:
@grantmcconnaughey So that means you will continue to maintain this package? As it currently stands, the package is not Django 4.0 compatible. There are even Django 3.2 issues.